client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -384,6 +395,21 @@ export const SeoForm = React.createClass( {
 		this.setState( { showPreview: false } );
 	},
 
+	getConflictingSeoPlugins( activePlugins ) {
+		const conflictingSeoPlugins = [
+			'Yoast SEO',
+			'Yoast SEO Premium',
+			'All In One SEO Pack',
+			'All in One SEO Pack Pro',
+		];
+
+		const plugins = flatMap( activePlugins, plugin => (
```

Comment:
why `flatMap()`? it doesn't exactly look like it would be different in practice here than just `map()` - am I missing something? _hint hint!_ an explanatory comment would help, something to show a little example of expected input and how it's transformed